books/free-programming-books-id.md

```diff
@@ -45,6 +45,7 @@
 * [Panduan Pengguna CodeIgniter Indonesia](https://codeigniter-id.github.io/user-guide/) - CodeIgniter Indonesia
 * [Tutorial CodeIgniter 3 & 4](https://www.petanikode.com/tutorial/codeigniter/) *(Dalam Proses)*
 * [Tutorial CodeIgniter 4](http://mfikri.com/artikel/tutorial-codeigniter4)
+* [Ebook Tutorial Codeigniter untuk Pemula](https://repository.bsi.ac.id/index.php/unduh/item/176695/Tutorial-Codeigniter-Untuk-Pemula.pdf)
```

Comment:
Please see [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) guidelines.

> - use the link with the most authoritative source (meaning the author's website is better than the editor's website, which is better than a third party website)
no file hosting services (this includes (but is not limited to) Dropbox and Google Drive links)

Use https://mfikri.com/artikel/tutorial-codeigniter as link